core/src/main/java/com/amplifyframework/core/model/AuthRule.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model;
+
+import androidx.core.util.ObjectsCompat;
+
+import java.util.Arrays;
+
+/**
+ * {@link AuthRule} is used define an authorization rule for who can access and operate against a
+ * {@link Model} or a {@link ModelField}.
+ *
+ * @see <a href="https://docs.amplify.aws/cli/graphql-transformer/directives#auth")>GraphQL Transformer @auth directive
```

Comment:
`")>` -- is the `)` supposed to be there?

Reply:
Good catch, removed!